src/main/scala/org/broadinstitute/dsde/firecloud/model/MethodEntity.scala

```diff
@@ -0,0 +1,33 @@
+package org.broadinstitute.dsde.firecloud.model
+
+import com.wordnik.swagger.annotations.{ApiModel, ApiModelProperty}
+import spray.json.DefaultJsonProtocol
+
+import scala.annotation.meta.field
+
+object MethodEntityJsonProtocol extends DefaultJsonProtocol {
+  implicit val impMethodEntity = jsonFormat10(MethodEntity)
+}
+
+@ApiModel(value = "Agora Method")
```

Comment:
Green team is not keen on extracting their model classes to a commons lib. Will discuss further, but is not likely. Will investigate swagger codegen to see if it can help.